### PR TITLE
cnv: Fix the wording around eviction strategy

### PR DESCRIPTION
**Pull request by fabiand**
Related to the fixes in https://github.com/openshift/openshift-docs/pull/74825/

### CODE REVIEW

**Comment by fabiand** [comment]
@sradco @avivtur please review

**Comment by sradco** [comment]
/lgtm

@ousleyp can you please review?

File: alerts/openshift-virtualization-operator/VMCannotBeEvicted.md [hunk 1]
```diff
@@ -49,5 +49,7 @@ issue:
 
 ## Mitigation
 
-Set the `evictionStrategy` of the VMI to `shutdown` or resolve the issue that
+Set the `evictionStrategy` of the VMI to `None` or resolve the issue that
 prevents the VMI from migrating.
+The `None` startegy shuts down VMs in case of node drains and pod
+evictions.
```
[the remark this answers is not in the document]
Comment (fabiand) [reply alerts/openshift-virtualization-operator/VMCannotBeEvicted.md]: Thanks!

I adjusted the wording according to the tech details.

**Pull request merged**